quiche/src/stream.rs

```diff
@@ -1144,11 +1158,16 @@ pub struct SendBuf {
 
     /// The error code received via STOP_SENDING.
     error: Option<u64>,
+
+    _bf: PhantomData<F>,
 }
 
-impl SendBuf {
+impl<F> SendBuf<F>
```

Comment:
If `BufFactory::Buf` is required to implement splitting and something like [chunks](https://doc.rust-lang.org/std/primitive.slice.html#method.chunks) (but splits into iterator of `BufFactory::Buf`s), I think we can enable sending without alloc+copy. `SendBuf::write` would take an application-provided `BufFactory::Buf`.